lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -229,9 +237,12 @@ def main():
                            supports_check_mode=True,
                            **module_kwargs)
 
-    # stack API introduced in 1.8.0
-    if not HAS_SHADE or (StrictVersion(shade.__version__) < StrictVersion('1.8.0')):
-        module.fail_json(msg='shade 1.8.0 or higher is required for this module')
+    # stack tag API introduced in 1.x.x
+    if not HAS_SHADE or (StrictVersion(shade.__version__)
+                         < StrictVersion('1.26.0')):
+        module.fail_json(
+            msg='shade 1.26.0 or higher is required for this module')
+
```

Comment:
This version downgrade is not acceptable. This module can call update_stack(), which is was added in 1.8.0, so you'd be breaking others by allowing an older version. Please don't change required versions.

Reply:
I'm not downgrading any version, just the opposite. Tag metadata support for heat was added in shade 1.26.0, which I understand won't be a downgrade. How would you suggest to tackle this? IMHO creating a stack and then upgrading would be slightly dirty.

Reply:
oh well, then it I suppose it's all right to rise the minimal version requirements for shade from 1.8.0 to 1.26.0?

Reply:
Oh wow, I *totally* misread this version as a LOWER version. I should not review before at least 2 cups of coffee! Apologies.

Yes, upgrading the version for a new feature is fine. However, we should only require the newer version if a tag is specified. Users not specifying a tag shouldn't be required to upgrade shade.